apps/examples/protobuf/protobuf_gtest/README.txt

```diff
@@ -1,5 +1,5 @@
-examples/protobuf_test
-^^^^^^^^^^^^^^^^^^^^^^
+examples/protobuf/protobuf_gtest
```

Comment:
If we put the test into the module folder, I think the app name is enough to have only the test scenario, without module name.
So I think `examples/protobuf/gtest` is enough.

Reply:
Right. I updated